pandas/tests/indexing/test_indexing_engines.py

```diff
@@ -0,0 +1,168 @@
+import numpy as np
+
+import pandas.util.testing as tm
+from pandas import compat
+from pandas._libs import algos as libalgos, index as libindex
+
+
```

Comment:
is there a way to just combine these into a single set of tests w/o regards to whether this is object / numeric. this is pretty duplicative set of tests

Reply:
Following the discussion below, I could just remove the tests for ObjectEngine.